componentified-todo-list/amirmahdi/to-do-list/src/index.js

```diff
@@ -1,57 +1,79 @@
-import React, { useState } from 'react';
+import React, { useState,useEffect } from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import Itemlist from './Itemlist';
 
 
 
 const App = () => {
+
   const [inputItems, setInputItems] = useState([]);
   const [inputValue, setInputValue] = useState("");
   const handleInputAdd = (event) => {
       event.preventDefault();
       if(inputValue.length===0){
         return;
       }else{
-      const id = (inputItems.length)?inputItems[inputItems.length - 1].id +1 :0;
-      setInputItems([...inputItems, { id: id, value: inputValue, complete:false }]);
       setInputValue("");
-      }
-  }
-  const handleRemove = (id)=>{
-         setInputItems(inputItems.filter(item => item.id !==id));
+        const body=JSON.stringify({
+          ownerId: 3,
+          title: inputValue,
+          checked : false
+        });
+        const headers={
+          'Content-Type': 'application/json'
         }
-  const handleAddToBottom = (id)=>{
-      let newArray=[...inputItems];
-      let objectIndex = newArray.findIndex(obj=>obj.id ===id);
-      if(objectIndex!==newArray.length-1){
-        let temp=newArray[objectIndex];
-        newArray[objectIndex]=newArray[objectIndex+1];
-        newArray[objectIndex+1]=temp;
-        setInputItems([...newArray]);
-      }
-    }
-  const handleAddToTop = (index) => {
-      let newArray=[...inputItems];
-      let objectIndex = newArray.findIndex(obj=>obj.id ===index);
-      if(objectIndex!==0){
-        let temp=newArray[objectIndex];
-        newArray[objectIndex]=newArray[objectIndex-1];
-        newArray[objectIndex-1]=temp;
-        setInputItems([...newArray]);
-      }
+       fetch('https://todolist.ehsan-rafee.ir/api/todolist',{
+        method:'POST',
+         body,
+         headers,
+      })
```

Comment:
__[HIGH]__  
How should react know that it should update UI? You should call `setInputItems` after calling `fetch`.